server/api/admin.go

```diff
@@ -138,3 +138,21 @@ func (h *adminHandler) UpdateWaitAsyncTime(w http.ResponseWriter, r *http.Reques
 	cluster.GetReplicationMode().UpdateMemberWaitAsyncTime(memberID)
 	h.rd.JSON(w, http.StatusOK, nil)
 }
+
+// @Tags admin
+// @Summary switch Service Middlewares including ServiceInfo, Audit and rate limit
+// @Param enable query string true "enable" Enums(true, false)
+// @Produce json
+// @Success 200 {string} string "Switching Service middleware is successful."
+// @Failure 400 {string} string "The input is invalid."
+// @Router /admin/service-middleware [POST]
+func (h *adminHandler) HanldeServiceMiddlewareSwitch(w http.ResponseWriter, r *http.Request) {
+	enableStr := r.URL.Query().Get("enable")
```

Comment:
How many switches do we need after supporting the audit and rate limit?

Reply:
three switches. A total service middleware switches, a rate limit switch, and an audit switch